stub/src/main/java/io/grpc/stub/ServerCalls.java

```diff
@@ -349,4 +349,31 @@ public void onCancel() {
     public void onComplete() {
     }
   }
+
+  public static void unimplementedStreamResponse(StreamObserver responseObserver) {
+    responseObserver.onError(Status.UNIMPLEMENTED.asException());
+  }
+
+  public static class UnimplementedStreamObserver<ReqT> implements StreamObserver<ReqT> {
```

Comment:
Let's put this behind a method to return an instance instead of making it public.